test/snippet/alphabet/cigar/cigar_operation.cpp

```diff
@@ -0,0 +1,15 @@
+#include <seqan3/alphabet/cigar/cigar.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    using namespace seqan3::literals;
+
+    seqan3::cigar::operation letter{'M'_cigar_operation};
+
+    letter.assign_char('D');
+    seqan3::debug_stream << letter << '\n'; // prints "D"
+
+    letter.assign_char('Z'); // Unknown strings are implicitly converted to M.
+    seqan3::debug_stream << letter << '\n'; // prints "M"
```

Comment:
Why are unknown strings are converted to `0P` and single letters to `M`?
```suggestion
    letter.assign_char('Z'); // Unknown characters are implicitly converted to M.
    seqan3::debug_stream << letter << '\n'; // prints "M"
```

Reply:
🤷

Reply:
Suggested change still open?

Reply:
Ah missed it :)